packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-pattern-details.js

```diff
@@ -82,5 +89,40 @@ export default function usePatternDetails( postType, postId ) {
 		/>
 	) : null;
 
-	return { title, description, footer };
+	const details = [];
```

Comment:
Curious to know why we're creating this array if there are only gonna be 1 or 0 elements in it?